test/e2e/lib/gutenberg/blocks/contact-info-block-component.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
```

Comment:
Left those comments for now as I'm not sure if I'll change these files soon, but will remove in a cleanup before taking this PR out of the draft state.